lib/wanda_web/controllers/error_json.ex

```diff
@@ -10,6 +10,17 @@ defmodule WandaWeb.ErrorJSON do
     }
   end
 
+  def render("404.json", %{reason: reason} = context) when not is_map_key(context, :status) do
```

Comment:
curiosity: what is this check for?

Reply:
some controllers might raise an error like this
```elixir
def show(conn, %{id: execution_id}) do
  execution = Executions.get_execution!(execution_id)

  render(conn, execution: execution)
end
```
and it bubbles up to the fallback controller in a map with `reason` being the error raised, `status` the status code to be returned, and entries.

I added it to keep things working, even though I personally prefer the `with` based approach returning {:error, :something}
```elixir
with {:ok, foo} <- Context.function(arg1, arg2) do
  render(conn, :foo, %{foo: foo})
end
```

We might consider streamlining implementations.

Reply:
This is the best I could get to keep also api redirector tests working c8ce80e07c2023f28020a9afc12f682ae99edb65